src/CoreWCF.NetTcp/src/CoreWCF/NetTcpSecurity.cs

```diff
@@ -12,18 +12,20 @@ public sealed partial class NetTcpSecurity
     {
         internal const SecurityMode DefaultMode = SecurityMode.Transport;
         private SecurityMode _mode;
+        MessageSecurityOverTcp _messageSecurity;
 
         public NetTcpSecurity()
-            : this(DefaultMode, new TcpTransportSecurity())
+            : this(DefaultMode, new TcpTransportSecurity(), new MessageSecurityOverTcp())
         {
         }
 
-        private NetTcpSecurity(SecurityMode mode, TcpTransportSecurity transportSecurity)
+        private NetTcpSecurity(SecurityMode mode, TcpTransportSecurity transportSecurity, MessageSecurityOverTcp messageSecurity)
         {
             Fx.Assert(SecurityModeHelper.IsDefined(mode), string.Format("Invalid SecurityMode value: {0}.", mode.ToString()));
 
             _mode = mode;
             Transport = transportSecurity ?? new TcpTransportSecurity();
+            Message = messageSecurity == null ? new MessageSecurityOverTcp() : messageSecurity;
         }
```

Comment:
I think this can be rewritten as `Message = messageSecurity ?? new MessageSecurityOverTcp()'`  
See [here](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator) for docs for null coalescing operators in C#